README.md

```diff
@@ -109,6 +115,23 @@ npm run test
         "wdio-lowlight": "terra wdio --disableSeleniumService=true --themes clinical-lowlight-theme"
         }
        ```
+7. To terminate a dev container:
+    - Use command line or Rancher/Docker Desktop OR
+    - Using Visual Studio Code
+      -  Select the Remote Explorer icon in the Activity Bar or View -> Command Palette and run **Remote Explorer: Focus on Containers View**
+      - Locate the **terra-core_devcontainer** or currently running dev container under "Dev Containers"
```

Comment:
*nitpick, terra-core can be updated here

Reply:
Yes, I missed that somehow!